### ISSUE TITLE
Генокрады не могут юзать армблейд в рубашке смирительной.

### ISSUE THREAD

**Issue by ghost** [GammaStation/Gamma-Station#315]
#### Подробное описание проблемы
Простую рубашку из ткани не может прорвать армблейд который может прорвать наручники.
#### Что должно было произойти
Футболка уничтожиться а вы свободны.
#### Что произошло на самом деле
Футболка на месте, вы не свободны.
#### Как повторить
Заковать себя в рубашку и попробывать юзнуть армблейд

**Comment by Miracler112** [comment]
Оно по идее и не должно рвать, руки ж скованы и сил всё равно не хватит её порвать. Однако скафандр генадия как раз сбрасывает рубашку. Но подожду что другие скажут, если это признают багом, то ладно

**Comment by Skippeer12** [comment]
Да, генокрад должен страдать, а то ишь в наручниках они могут убивать людей, вообще уже офигели.
Я бы ещё выпилил им возможность снимать рубаху скафандрами, но чёт лень
Копите поинты на биогрейд или скафандр.
